Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "b8fc738d4e14598e36269ce0d849489f95562ba047e5663caca9ac02550893ef"
+            "sha256": "44ae793965dc2b6ec17f0435a388846248b8a703cf857470b66af84227535950"
```

Comment:
It looks like dependencies were also updated with newer versions, I guess as a side-effect of removing the `v` line?

Is it possible to remove the package without reinstalling the other packages (and thereby changing lockfile versions?)

> Sidenote: I think we should switch our `pipfile` to use explicit versions, instead of '*', but thats for another pr

Reply:
Possible... a bit hacky (would have to manually edit the lock file).

The way I see it, it's only minor version upgrades which *should* be safe since being loose on these is pretty standard. Carefully reviewing every minor version upgrade of a dependency isn't usually considered worthwhile, the benefit of the lockfile currently is just that we see & sort of control *when* they change.

None of the changes are in direct runtime dependencies.

My plan is to make things a bit more explicit along with migrating to poetry in the near future.